xzes40/transformer/src/lib.cpp

```diff
@@ -138,3 +152,17 @@ std::string xzes::_hash( std::string input )
 
     return ret.str();
 }
+
+// ----------------------------------------------------------------------------
+// Returns if a request sent via unix socket is valid.
+// ----------------------------------------------------------------------------
+int xzes::valid_request(char * b)
```

Comment:
This method is very naive but is supposed to verify that the input is o the format `job-id,input-xml,input-xsl`.